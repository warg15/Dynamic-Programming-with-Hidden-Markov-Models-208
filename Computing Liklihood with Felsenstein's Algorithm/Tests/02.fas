>Yellow-throated-sandgrouse
AACTACTACCGGATGAACAATGCTCCAAAACATTACGGTTCCTTGTACTAGCAGACGAAAAAAACGTTTTAACAGCAGGCAGGGCTATTGATAACTGGTGATTGAATGCAAGTTGCCGCATCAGTACACTAACAAGTACTAGTTGCGGGTGAGGAAGAACGACTTCCGCAGTAACTACTAACTAGTGAGACAGGTAGATA
>Great-cormorant
AACTACTGCCGGATGAACAGTCCTCCAGAACATTACTTTTCCCTGTACTAGTAGATGAAAAAGACATTTTAACAGCAGGCTGAGCTACTGAGAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAGCTACTAACTAGTGAGATAAACAGATA
>Carmine-bee-eater
TTCTACTACCGGATGAGCAGTGCTCCAAAACATTACGTTACCTTGTGATAGAAGACGAAAAAAACATTCTAACAGCAGACAGAATTACTGACACCTGGTAGCTGAATGCGAGCTATCGTATTAGTACATTAACAAGTACTACTCGCGGGTCAAGAGGAATGACCTCCGCCTTAGCTGCTAACTAGTAAGACAGACAGATA
>Rifleman
AACTACTACCGGATAAACATTGATCCAAAACATTATGTTTCCTTGTACGAGATGACGAAAGAGACATCTTAACAACAGGCAAAGCTACTGATAATTGCTAGTAGAACGCAAGTTATCGCATTAGTACATTAACAAGTAATACTTGCGGGTCAGGAAGGGCGACTCCCGCCGTAGCTACTAGCTGGTGAGACAGATGGATA
>Pekin-duck
AGCTACTACCAGTTAAACAGTGCTTCAAAACATTACGCTTCCTCGTGCTAGCAGACAAAAAAGATATTTTGGCAGCAGGCAGAGCTACTAATAAATGATGGTTGAATGCAACTTATCACATTGGTACATTAGCAAGCACTACTTGCGAGTCAGAAAGAACGACCTCCGCCATAGCTACTAACTCGTGAGACCGACACATA
>Emperor-penguin
AACTACTACCGGACGAACAGTCCTCCAAAACATTATTTTTCCTCGTACTAGTAGACGAAAAAGACATTCTAACAGCAGGCAGAGCTAATGATAGTTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTAAGGAAGAACGACTTCCGCCGTAGCTACTAACTAATGAGACAAACAGATA
>Budgerigar
AACTACTACCGGATGAACAGTGCTCCAAAACATCATGTTTCCTTGTAATAGAAGACTGAAAAGACATCTTTACAGCAGGCAGAGCTAGTGATAATTAATAGTTGAATGCAAGTTGCCGCATTAGCACATTAACTAGTATTATTCGTGGGCCAGGAAGGACGACTTCCACCGTAGCTACTAACTAGCGAGACAGACAGATA
>Turkey-vulture
AACTACTACCGGATGAACAGTGCTCCAAAACATTACGTTTCCTTGCACTAGAAGACGAAAAAGACATTTTAACAGCAGGCAGGGCTACTGATAATTTGTAGTTGAATGCAAGTTGCCGCATTCGTACATTAACAAGTACTACTTGCGGGTCAGGGAACATGACTTCCGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Red-throated-loon
AACTATTACTGGATGAACAGTCCTCCAAAACATTACTTTTCCTTGTACTAGCAGACGAAAAAGACATTCTAACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAATTGCCACATTAGTACATTAACAAGTACTACTTGCGGGTCAGGAAGAACGACTTTCGCCGTAGCTACTAACTAGTGGGACAGACAGATA
>Dalmatian-pelican
AACTACTACCGGATGAACAGTCCTCCAAAACATTACTTTTCCTTGTACTAGTGGACGAAAAAGACATTCTCACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Common-cuckoo
AACTACTACCGGATGAACAGTGCTCCAAGACATTACCTTTCCTTGTACTAGCAGACGCAAAAGACATTTTAACAGCAGGCAGAGCTACTGATAATTGGTAGTAGAATGCCAGTTGCCGCTTTAGTACATTAACCAGTACTACGTGCGGGTCAGGTAGAACGACTTTCGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Chuck-wills-widow
AACTACTACCGGAGGAACAGTGATCCAAAACATTACGTTTCCTTGTATCAGCAGACGGAAAAAACATTTTACCGGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAGTTGCAGCATTAGTACATTAACAACTACTACTTTCGGGTCAGGAAGAACGACTTCCGCCGTAGCTACTAACGAGTGAGACCGACAGATA
>American-flamingo
AACTACTACCGGATGAATAGTGCTCCAAAACATTACGGTTCGTTGTGGTAGCAGACGAAAAAGACATTTTAACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAACGCAAGTTGCCGCATTAGTACGTTAGCAAGTACTACTTGCGGGTCAGGAAGAACCAAATCCGCCGTAGCTATTAACTAGTGAGAAAGACAGATA
>MacQueens-bustard
AACTACTACCGGATGAACCTTACTCCGAGACATTACATTTCCTTGTACTAGCAGGCGAAAAAGACATTTTAACAGCAGGCAGGGCTACTGATAATTGCTATTTGAATGCAAGTTGCCGCATTAGTGCATTAACAAGTAGTACTTACAGTTCAGGAAGAACAACTTCCGCCGTAGCTACTGGCAAGTGAGACAGACAGATA
>White-tailed-tropicbird
AACTACTACCGGATGAACAGTGCTCCAAAAGATTACTTTTCCTTGTACTAGCAGACGAAAAAGACATTTTAACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACGTTAACAGGTATTACTTGCGGGTCAGGAAGAACGACGTCCGCCGTAGTTACGAACTAGTGAGACAGACAGATA
>Adelie-penguin
AACTACTACCGGACGAACAGTCCTCCAAAACATTATTTTTCCTCGTACTAGTCGACGAAAAAGACATTCTAACAGCAGGCAGAGCTAATGATAGTTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTAAGGAAGAACGACTTCCGCGGTAGCTACTAACTAATGAGACAAACAGATA
>Peregrine-falcon
AACTGCTACTGGATAAACAGTGCACCGAAACATTACGTTTCCTTATACTAGAAGACGAAAAAGACATTTTAACAGCAGGCAGAGTTACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTGCATTAACAAGTACTACTTGCGGGTCAGGAAGAATGGCTTCCGCCGTAGTTACTAACTAGTGAGACAGACAGGTA
>Chicken
AGCTACTAGCGGGTGAACTATGTTCCAACACATTACGTCTCCTGGTGCGAGCGGACGAAAAAGATATTCTGGCAGCAGGCATAGCTATGGATAATAAGTAGTTAAATGCGAATTATTGCATTAGTACCTTAGCAACCACTACTTGCGGATCAGGAAGAGCGACATCTGCCATTGCTTCTAAATAGTGAGACAGGCGCACA
>Rhinoceros-hornbill
AACTAGTACCGGATTAACAGTGCTCCAAAGCACTACGTTTCCTTGTACTAGAAGACGAAAAAGACATTCTCACAGCCGGTATTGCTACTGATAATTGTTAGTTGAATGCAAGTTGCCGTATTAGGACAGTAACAAGTACTACGTCCGGGTCAGGAAGAATGACTTCCGCCTTAACTACTAACTAGTAAGACAGACAAATA
>Cuckoo-roller
AAATACTGGCGGATGAACAGTGGTCCAAAACATTACGTTTCCTTGTACTTGAAGACGAAAAAAACATTTTAACAGCAGGCAGAGCTACTAATAATTGGTAGTTGAGTGCGAGTCGCCGCATTAGTACATTAATAAGTACTACTTGCGGGTCAGGAAGAATGACTTCCGCCTTAGCTACTAAATAGTAAGACAGACAGATG
>Great-crested-grebe
AACTACTATCGGATGAACAGTACTCCAGAACATTCCGTTTCCTTGTACCAGCAGACGAAAAAGACATTTTAACAGCAGGCAGAGCTACTGATAATTGGTAGCTGAATGCAAGTTGCCGCATTAGTACATTAGTAAGTAATACTTGCGGGTCAGGAAGAACCACATCCGCCGTGGCTACTAACTAGGGAGGCAGACAGATA
>Crested-ibis
AACTACTACCGGATGAACAGTCCTCCAAAACATTACTTTTCCTTGTACTAGTAGACGAAAAAGACATTCACACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTGAGGAATAACGACTTTCGCCGTAGCTACTAGCTAGTGAGACAGACAGATA
>Sunbittern
AACTACTAGTGGATGAACAGTGCTCCAAGACGTTACTTTTCCTTGTACTAGCAGACGAAAAAGACATTTTAACAGCAGGCAAAGCTACTGATAATTGGTAGTTGAATGTAAGTTGCCGCATTAGTACATAAATAAGTACTACTTGCCGGTCAGGAAGGACGAATTCCGCCGTAACTACTAACTAGTAAGACAGACAGATG
>Golden-collared-manakin
GACTATTATCGGATGAACAGTGATCCAAAACATTACGTTTCCTCGTACTAGAAGACGAAAAAGACATTTTATCAACAGGCAAAACTACTAATAATTGGTAGTTGAAGGCAAGTTGTCGCATTAGTACATTAACAAATGCTACCTGCAGGTCAGGAAGGGCGACTGCCGCCGTAGCTACTAACTAGTGAGACAGACGGATA
>Red-legged-seriema
AACTACTATCCGGTGAACAGTGCTCCAAAACATTCCGTTTGCTTGTACTAGAAGACAAAAAAGACTTTACAACAGCAGGCAGAGCTACTGATAATTGCTAGTTGAATGCAAGTTGCCGCATCAGTACATTAACAAGTGCTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAACTGCTAACCAGTGAGACGGACAGGTA
>Brown-mesite
AACTACTACCGGATGATCTGTGCTACAAAACATTATGTTTCCCCGAACTAGCAGACGAAGAAGACATTTTAACTGCAGGCAGAGCTAATGATGATTGGTAGTTGAATGCAAGTTGTCGCATTAGTACATTAACAAGTACTAGTTGCAGGCCAGGAAGGACGAATTCCGCCGTAGCTACTAACTAGAGAGACAGACAGATA
>Kea
AACTACTACCGGATGAACAGTGCTCCAAAACATCATGTCTCCTTGTACTAGAAGGCGAAAAAGACATTTTTACAGCAGGCAGAGCTAGTGATAAATGATAGCTGAATGCAAGTTGCCGCATTAGCACATTAACAAGCATTACTTGTGGGTCAGGAAGAACGACTTCAGCCGTAGCTACTAACTAGTGAGACAGACAGATG
>White-throated-tinamou
AACTACTACCGGGTGAACCGTGATTCAAAACACTACGTCTCTGCGCTCTGGCAGACGAAAATAAAATTTTAACAGCAAGCAGCATCTGTAATAATTGGTAGTTGAATATAAATTGACGCACTCGTACTTTAGTAGGGACTACTTGCGGGTCAAAAAGAACGTCTTCCTGCGGAGTTACTGAATAGTGAGATAGACAGACA
>American-crow
AACTACTACCGGATGAACAGTGATCCAAAACATTTCGTCTCTTTGTACGAGAAGACGAAAAAAACATTTTAAGAAGAGGCAAAGCTATTGATAATTGGTAATTGAATGCAAGTTGTCGCATTAGTACATTAACAACTACCACTTGCGGGTCAGGAAGGGCGACTCCCGCCGTAGCTACTAACTGGTGAGACAGACGGATA
>Turkey
AGCTACTTGCGGGTGAACAGTGTTCCAAAACATTAAGTTTCCTGGTGCGAGCGGATGAAAAAGATATTCTGGCAGCCGGCATAGCCACGGATAATAAGTAGTTAAATGCGAATTATTGCATTAGTACCTTAGCAACCACTACTTGCGGATCAGGAAGAACGACACCCGCCACAGCTTCTAAATAGTGGGACAAGCACACA
>Red-crested-turaco
AACTACTATCGGATGAACGGTGCTCCAAAACATTTCGTTTCCTTGTACTACCAGGCGAAAAAGACATTATAACAGCAGGTAGAGCTACTGATAACTGCTAGTTGAATGCAAGTTGCCGTATTAGTACATTAACAAATACTACTTGCGGGTCAGGAAGAACGACTTCCGCTGTAGCTGCTAACTAGTGAGACAGACAGATA
>Hoatzin
AACTACTACCGGTTGAACAGTGCTCCAAAACATTACGTTTCCTTGTACTAGCAGGCGAAAAAGACATTTTAACAGCAGGCAGAGCTACTGACAATTGGTAGTTGAAAGCAAGTTGCCGCATTAGTACACTAACAAGTACTACTTGCGGGTCAGGGAGAACGACTTTCGCCGTAGCTACTAACTGGTGAGGCAGACAGATA
>Common-ostrich
ATCTACTACCGGATGAACCGTGTTCTAAAACATTACATCTCCTTGTGCTGGCAGACGAAAATGACATTTTACCAGCAGGCAGCGTTAATAATCATCGGTAGTTGAGTGCAAATTGTCGCATTAGTACGTTAGCAAGGACTAGTTGAGGGTCAGGTAGAACATCTTCCGTCGGAGTTACTGAATACTGAGATAGACGGACA
>Zebra-finch
AACTACTACCAGATGAACAATGATCCAAAACATTTCGTTTCCTTACACTAGAAGACGAAAAACACATTTTAGGAAGAGGCAAAGATATTGATAATTGATAGTTGAATGCGAGTTGTCGCATTAGCACATTAACAACTACTACTTGCGGGTCAGGAAGGGCGACTCCCGCCGTAACTACTAACTAGTGAGACAGACGGATT
>Little-egret
AACTACTACCGGATGAACAGTCTTCCAAAACATTAGTTTTCCTTGTACTAGTCGGCGAAAAAGACATTCTCACAGCAGGCAGAGCTACTGATAATTAGTAGTTGAATGCAAGTTGCCGTATTAGTACATTAATAAGTACTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAGGTACTAACTAGTTAGACAAACAGCTA
>White-tailed-eagle
AACCACTACCGGATGAGCGGTGCTCCAAAACATTGCGTTTCCTTGTGCTAGAGGACAAAAAAGACATTTTAGCAGCAGGCAGAGCCACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGAGGGTCAGTAAAAATGTCTTCGGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Chimney-swift
AACTACTACCGGATGAACAGTGCTCCAAAACATTACGTTTTCTTGTACTGGCGGACGGAAAAGACATTTTAGCAGCAGGCAGAGCTACCGATAATTAATAGTTGAATAAAAGCTGCCGCATTCGTACATTATCAAGTACTACTTGCGGGTCAGGAAGAACTATTTCCGCCGTAGATACTAACTAGTGAGACAGGCAGATC
>Annas-hummingbird
AACTACTACCAGAGGAACAGTGCTCCAAAACATTGCATTTTCTTGTACTAACGGACAGAAAAGACATTTTAACAGCAGGCAGAGCTACTGATAATTAGTGGTTGTATGCAAGTTGCCGCATTAGCAGATTATAAAGTACTACTCGCGGGTCAGAAAAAACGACTTCCGCCGTAGCTACTAACGAGAGAGACAGCCAGATA
>Northern-fulmar
AACTACTACCGGATGAACAGTCCTCCAAAACATTACTTTTCCTTGTACTAGTAGACGAAAAAGACATTCTAACAGCAGCCCGAGCTACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGCACATTAACAAGTACTACTTACGGGTCAGGAAGAACGACTTCCGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Pigeon
ACCTAGTATCGGACGAACAGTGCTCCAAGACATTACGTTTCCTTGTACTAGCAGACGATAAAAACATTTTAATAGCAGGCAGAGCTACTGATAATTGGTAATAGAATGCAAATTGCCGCATTAGTACATCAGCAAGTATTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAGGTACTAACTAATGAGACAGACAGATA
>Medium-ground-finch
AACTACTACCAAATGAAGAATGATCCAAAACATTTCGTTTCCTTATGCTAGAAGAGGAAAATGACATTTTAAAAAGAGGCAAAGATATTGATAATTGGTAGTTGAATGCGAGTTGTCGCATTAGTACATTAACAACTACTACTTGCGAGTCAGGAAGGGCGACTTCCGCCGTAACTACTAACTAGTGGGACAGACGGATA
>Speckled-mousebird
AACTACTACTAGATAAACAGAGCTCCAAAACATTGCGTTTCCTTGTACTAGAAGACGAAAAAGAAATTTGAACAGCAGGCAGAGCTACTGATAGATGGTGGTTGAATGCAAGTTACCGCATTAGTACATTAACAAGTACTGCTTGCGGGTCAGGAAGAATGACTTCCGAATTAGCTACTATCTAGTGAGAGACACAGATA
>Bar-tailed-trogon
AACTACTACCGGATAAACAGTGCCCCAAAACATTAAGTTCCCTTGTACTAGAAGACGCAAAAGCTACTTTAGCAGCCGACAGAGCTACTGATAATTGGTGGTTGAATGTAAGTTGCTTCATTAGTGCATTATGAAATACCACTTGCGGGTTAGGAAGAATGACTTCCGCCTTAGCTACTACCTAGTAAGACAGACAGCTA
>Barn-owl
AACTACTACCGGATGAACAGTGCTCCAAGGCATTACGTTTCCTTGGACTAGAAGACGAGAAAGACATTTTAAGATCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCAAATTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTCAGGAAGAATGACTTCCGCCGTAGCTACTAACTAGTGAGGCAAACGGATA
>Killdeer
AACTACTACCGGATGAACAGTGCTCCAAATCATTACGCTTCCTTGTACTAGCAGACGAAAAAGAGATTTTGACAGTATTCAAAGTTACTGATGATTGGTAGTTGAATGTAAGTTGCCGCATTAGTACATTAGCAAATGCTACTGGCGGGTCAGGGAGAACGACTTCCGCCGTAACTACTAACTAGTGAGACAGCCAGCTA
>Grey-crowned-crane
AACTACTACCGGATGATCAGGGCTCCAAAACATTACGCTTCCTTGTACTAGCAGATGAGAAAGACATTTCAACAGCAGGCAGAGCTACTGATAATTGGTAGTTGAATGCACGTTGCCGCATTAGTACATTAACAAGTACTACTTGCGGGTCAGGAAGAACGACTTCCGCCGTAGCTACTAACTAGTGAGACAGACAGATA
>Downy-woodpecker
AACCACTACCGGGTGAACAGTGCTTCAACACATTACGTTTGCTTGTAATAGAAAACGAAAAAGGCATTTTAACAGCAGGCAGAAGTACTAATAATTCGTAATTGAATGCAAGTTGCCGCAATAGTACGTTCACAAGTACTACTCGCGGATCAGGAATAAGGACTTCCGTGTCAACTACTAACTAGTGAGACAGACAGGTA
>Bald-eagle
AACCACTACCGGATGAGCGGTGCTCCAAAACATTGCGTTTCCTTGTGCTAGAGGACAAAAAAGACATTTTAGCAGCAGGCAGAGCCACTGATAATTGGTAGTTGAATGCAAGTTGCCGCATTAGTACATTAACAAGTACTACTTGAGGGTCAGTAAAAATGTCTTCGGCCGTAGCTACTAACTAGTGAGACAGACAGATA
