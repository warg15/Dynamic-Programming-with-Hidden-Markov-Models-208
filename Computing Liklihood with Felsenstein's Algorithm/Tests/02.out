-3437.6307395821927
